assets/css/components/_table-search-result.scss

```diff
@@ -0,0 +1,95 @@
+.table-search-result {
+  $self: table;
+
+  @apply mx-0 min-w-full;
+
+  .#{ $self }__column {
+    &--index {
+      flex-basis: 4%;
+    }
+
+    &--keyword {
+      flex-basis: 20%;
+    }
+
+    &--status {
+      flex-basis: 13%;
+    }
+
+    &--top-advertiser-count {
+      flex-basis: 12%;
+    }
+
+    &--other-advertiser-count {
+      flex-basis: 12%;
+    }
+
+    &--page-results {
+      flex-basis: 12%;
+    }
+
+    &--total-page-results {
+      flex-basis: 14%;
+    }
+
+    &--cta {
+      @apply invisible;
+
+      flex-basis: 13%;
+    }
+  }
+
+  .#{ $self }__item {
```

Comment:
Wondering if we can merge these column width styling together

```scss
.#{ $self }__column,
.#{ $self }__item {
  ...
}
```

and for custom styling, we can just separate it.